### PR TITLE
Fix uninstall script

### LINKED ISSUE #2378: Uninstall script fails
Here's the output after uninstalling this extension:
```
[2020-02-18 13:13:38.280] [sharedprocess] [info] vscode-icons-team.vscode-icons 10.0.0 Running post uninstall script
[2020-02-18 13:13:38.396] [sharedprocess] [error] vscode-icons-team.vscode-icons 10.0.0 post-uninstall internal/modules/cjs/loader.js:630
    throw err;
    ^

Error: Cannot find module '/Users/netweb/.vscode/extensions/vscode-icons-team.vscode-icons-10.0.0/out/src/uninstall.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:627:15)
    at Function.Module._load (internal/modules/cjs/loader.js:531:27)
    at Function.Module.runMain (internal/modules/cjs/loader.js:850:10)
    at internal/main/run_main_module.js:17:11 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

[2020-02-18 13:13:38.406] [sharedprocess] [error] vscode-icons-team.vscode-icons 10.0.0 Failed to run post uninstall script: post-uninstall process exited with code 1
[2020-02-18 13:13:38.511] [sharedprocess] [info] Deleted from disk vscode-icons-team.vscode-icons /Users/netweb/.vscode/extensions/vscode-icons-team.vscode-icons-10.0.0
```

Installed the extension again and the path and file do not exist:

- `/Users/netweb/.vscode/extensions/vscode-icons-team.vscode-icons-10.0.0/out/src/uninstall.js`

Only `dist`, `icons`, and `images` folders in the `/Users/netweb/.vscode/extensions/vscode-icons-team.vscode-icons-10.0.0/` path and the various package.json files in the root

### PR DESCRIPTION
**Pull request by JimiC**
_**Fixes #2378**_

**Changes proposed:**

- [x] Fix

### CODE REVIEW

**Comment by JimiC** [comment]
This PR introduces the creation of an `uninstall` bundled script. It's a separate script than the main extension bundled script.
I also tested in live-action.

File: src/iconsManifest/iconsGenerator.ts [hunk 1]
```diff
@@ -156,6 +171,20 @@ export class IconsGenerator implements models.IIconsGenerator {
...
+      // update 'vscode:uninstall' script
+      predicate = (line: string): boolean =>
+        line.includes('"vscode:uninstall"');
+      lineIndex = rawText.findIndex(predicate);
+      if (lineIndex > -1) {
+        rawText[lineIndex] = rawText[lineIndex].replace(
+          manifest.scripts['vscode:uninstall'],
+          uninstallEntryPath,
+        );
+        // eslint-disable-next-line no-console
```
Comment (jens1o) [inline src/iconsManifest/iconsGenerator.ts]: I guess after https://github.com/vscode-icons/vscode-icons/pull/2402 has been merged, this line can be removed, can't it?

Comment (JimiC) [reply src/iconsManifest/iconsGenerator.ts]: Of course not. This is what dynamically changes the path to the script.

Comment (JimiC) [reply src/iconsManifest/iconsGenerator.ts]: If you mean the rule, I guess so.

Comment (jens1o) [reply src/iconsManifest/iconsGenerator.ts]: Yeah, I meant the rule. I'm sorry for the misunderstanding. :/

Comment (JimiC) [reply src/iconsManifest/iconsGenerator.ts]: It's OK. I tend nowadays to fast read everything due to the situation and missing meanings.

**Pull request merged**